Flick/Controllers/EditProfileViewController.swift

```diff
@@ -360,12 +404,13 @@ extension EditProfileViewController:  UIImagePickerControllerDelegate, UINavigat
         let resizedImage = image.resize(toSize: profileImageSize, scale: UIScreen.main.scale)
         profileImageView.image = resizedImage
         profileSelectionModalView.removeFromSuperview()
+        didChangeProfilePic = true
```

Comment:
similar to what I said before, if we just send over the same image, it doesn't really change anything on front end of backend in terms of final result. that can save us some code on the front end. up to your discretion though

Reply:
i thought i would do it so backend don't have to reupload the image, but might not be necessary if we move to use only base64 and no storing